external-crates/move/crates/move-vm-runtime/src/interpreter.rs

```diff
@@ -234,16 +232,10 @@ impl Interpreter {
                         .map_err(|e| set_err_info!(current_frame, e))?;
 
                     if func.is_native() {
-                        self.call_native(
-                            &resolver,
-                            gas_meter,
-                            extensions,
-                            func,
-                            vec![],
-                        )?;
+                        self.call_native(&resolver, gas_meter, extensions, func.clone(), vec![])?;
```

Comment:
Confused about this `clone` that was added. Is it still needed?

Reply:
yes, because the compiler said so 🤷

Reply:
Can you share a paste with the error you get when it's not there?  This is cloning an `Arc` so it's maybe not the end of the world, but I think it's important to understand what's going on here, even if we can't avoid it, especially given this is part of the interpreter.

Reply:
```
365 |     fn call_native_impl(
    |        ---------------- in this method
...
370 |         function: Arc<Function>,
    |                   ^^^^^^^^^^^^^ this parameter takes ownership of the value
help: clone the value to increment its reference count
    |
352 |         self.call_native_impl(resolver, gas_meter, extensions, function.clone(), ty_args)
    |                                                                        ++++++++
```